soccer/planning/low_level/TrapezoidalMotion.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
```

Comment:
TODO dedup with `motion/TrapezoidalMotion.hpp`, which is broken for some of the cases this needs to handle.

Regardless, this should live in planning rather than control.